### PR TITLE
[Console] Move back root exception to stack trace in verbose mode

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #29232
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by chalasr** [comment]
3.4 totally misses the root exception when it's a built-in console `ExceptionInterface` instance, no matter the verbosity (https://github.com/symfony/symfony/pull/24131). Should we consider it a bug?

File: src/Symfony/Component/Console/Application.php [hunk 1]
```diff
@@ -762,7 +762,7 @@ protected function doRenderException(\Exception $e, OutputInterface $output)
             }
 
             $messages = array();
-            if (!$e instanceof ExceptionInterface || OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity()) {
+            if (!$e instanceof ExceptionInterface && OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
```
Comment (lyrixx) [inline src/Symfony/Component/Console/Application.php]: the superior to condition seems weird, isn't?

Comment (chalasr) [reply src/Symfony/Component/Console/Application.php]: reverted

**Review by lyrixx — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**